pre_install_report/library/pre_install_utils.py

```diff
@@ -77,12 +77,12 @@ def do_cmd(self, test_cmd):
 
             self.logger.info("cmd {} rc = 0".format(test_cmd))
             self.logger.debug("cmd {} rc = 0 response {}".format(test_cmd, str(data)))
-            return 0
+            return 0, str(data)
         except CalledProcessError as e:
             data = e.output
             self.logger.error("do_cmd " + ' rc = ' + str(e.returncode) + test_cmd +
                               ' data = ' + str(data))
-            return e.returncode
+            return e.returncode, str(data)
```

Comment:
Same comment as above.

Reply:
If rc is non zero, I now incorporate the message returned into the generated report.  This approach can be expanded to other kubectl calls as needed..

Reply:
so is the exception path the only time the returned message is incorporated into the generated report?   Because for the try path it seemed your response was the message could be incorporated into the report if needed.

Reply:
The exception path is the only one where message is currently incorporated.  But this could change in the future.